inc/meta-box-models.php

```diff
@@ -368,18 +368,17 @@ public function validate_datetime($field, $post_ID) {
 		}
 		if ( $field['type'] == 'date' ) {
 			$date = DateTime::createFromFormat('F j Y', $data[$field['taxonomy']]);
-		} elseif ( $field['type'] == 'time' ) {
-			if ( isset( $_POST[$field['taxonomy'] . '_timezone'] ) )
-				date_default_timezone_set( $_POST[$field['taxonomy'] . '_timezone'][0] );
-			$date = DateTime::createFromFormat('h:ia T', $data[$field['taxonomy']]);
-		} elseif ( $field['type'] == 'datetime' ) {
+		} else {
 			if ( isset( $_POST[$field['taxonomy'] . '_timezone'] ) )
 				date_default_timezone_set( $_POST[$field['taxonomy'] . '_timezone'][0] );
-			$date = DateTime::createFromFormat('F j Y h:ia T', $data[$field['taxonomy']]);
+			if ( $field['type'] == 'time' ) {
+				$date = DateTime::createFromFormat('h:ia T', $data[$field['taxonomy']]);
+			} elseif ( $field['type'] == 'datetime' ) {
+				$date = DateTime::createFromFormat('F j Y h:ia T', $data[$field['taxonomy']]);
+			}
 		}
-		if ( $date ) {
+		if ( $date )
 			$this->Callbacks->date( $post_ID, $field['taxonomy'], $multiple = $field['multiple'], $data, null );
-		}
```

Comment:
Sorry, one thing before we merge: please reinstate these braces, per the [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/php/#brace-style) (and, I think, CFPB's preferred standards, though it may not be documented anywhere).